packages/qwik/src/core/render/jsx/jsx-runtime.ts

```diff
@@ -285,7 +285,9 @@ const validateJSXNode = (node: JSXNode) => {
             `The JSX element <${type}> can not have both 'dangerouslySetInnerHTML' and children.`,
             node
           );
-          logError(err);
+
+          // just throw it as an error because we do not need to crash the app
```

Comment:
So logError crashes the app but throwing doesn't? If so, fix logError instead, no?

Reply:
I'd have done this but because `logError` is being used in multiple places, and idk if it should be intentional to halt the app, that's why i just made it like this (for now) until we decide later what should halt the dev server and what should not.

Reply:
well, nothing should halt the devserver ever. But invalid qwik code should definitely break SSR.
For tests, logError should always throw, except that it doesn't have to log when the test is expecting an error.
So maybe we could have something like `invokeFn` called `quietErrorsInvoke()`, that is used around tests that expecte errors?